opennms-webapp/src/main/java/org/opennms/web/admin/roles/AdminRoleServlet.java

```diff
@@ -274,18 +274,18 @@ private class SaveDetailsAction implements Action {
         @Override
         public String execute(HttpServletRequest request, HttpServletResponse response) throws ServletException {
             if (request.getParameter("save") != null) {
-                String roleName = request.getParameter("role");
+                String roleName = WebSecurityUtils.sanitizeString(request.getParameter("role"));
                 WebRole role = getRoleManager().getRole(roleName);
                 if (role == null) {
                     // this is a new role so create a new on and add it to the roleManager
                     role = getRoleManager().createRole();
                 }
-                role.setName(request.getParameter("roleName"));
+                role.setName(WebSecurityUtils.sanitizeString(request.getParameter("roleName")));
                 role.setDefaultUser(getUserManager().getUser(request.getParameter("roleUser")));
```

Comment:
shouldn't roleUser and roleGroup also be sanitized? I'm not really familiar with this workflow, I'm guessing thos are not avaliable for XSS but I'm just curious.

Reply:
roleUser and roleGroup are populated via 'Supervisor' and 'Membership Group' in the web UI, which are both drop downs. I tried to use the description field as well to XSS, but it doesn't seem to allow it. 
![image](https://user-images.githubusercontent.com/93448083/184424000-37a52890-23b7-4ce0-975a-e7551ba4719c.png)